atomic_reactor/plugins/post_pulp_pull.py

```diff
@@ -84,4 +84,9 @@ def run(self):
                        image_id)
         self.workflow.builder.image_id = image_id
 
-        return image_id
+        media_types = ['application/json']
```

Comment:
if neither pulp_push nor pulp_sync ran, then pulp_pull won't run either.
if pulp_push or pulp_sync ran, the first element in media_types will be ['application/json'] - at least until we handle "manifest lists, with compatibility formats back to V2schema1, without V1" which is more forward-proofing than I think we need to be concerned with at this time.

In short, returning ['application/json'] always appears to be correct unless I'm misreading https://projects.engineering.redhat.com/browse/OSBS-3874